Feature/integrate ts upload control

PenghaiZhang opened the pull request:
This PR adds the integration between Kaltura Plugin and the new Upload control. I added a new Typescript project where I created a function to use the new control. Next, in order to make the function accessible from server, I added a plain JS file to add the function to global JS object `window` which server can access. 

I have not added any test or lint yet, considering this TS project is so simple.

PenghaiZhang commented on Kaltura/com.tle.web.wizard.controls.kaltura/src/com/tle/web/controls/kaltura/KalturaHandler.java:
```diff
@@ -696,17 +691,17 @@ public void searchClicked(SectionInfo info)
+		public KalturaUploadInfo(String mediaType, String id)
 		{
 			this.mediaType = mediaType;
-			this.entryId = entryId;
```
The name of this field is `id` now rather than `entryId`.

edalex-ian commented:
Heya @PenghaiZhang 

As discussed, please remove any cruft that is there to support the old KCW widget.

PenghaiZhang commented:
> Heya @PenghaiZhang
> 
> As discussed, please remove any cruft that is there to support the old KCW widget.

Hi Ian, I also removed code related to the UI where users can attach previously uploaded files because we only support uploading one file now.

Here is a full video after I removed the code.

https://user-images.githubusercontent.com/47203811/122692803-71adf100-d27a-11eb-8536-0157a7658074.mp4

edalex-ian commented on Kaltura/com.tle.web.wizard.controls.kaltura/src/com/tle/web/controls/kaltura/KalturaHandler.java:
```diff
@@ -526,16 +465,15 @@ protected SectionRenderable renderAdd(RenderContext context, DialogRenderOptions
+//					renderable = renderUploads(context, renderOptions);
+//				}
+//				else
+//				{
```
Should probably remove these commented out sections?

PenghaiZhang replied:
Ah yeah thanks! Done.

edalex-ian commented on Kaltura/com.tle.web.wizard.controls.kaltura/src/com/tle/web/controls/kaltura/KalturaHandler.java:
```diff
@@ -526,16 +465,15 @@ protected SectionRenderable renderAdd(RenderContext context, DialogRenderOptions
 					setupKalturaKcw(context);
 					renderable = renderContribution();
-				}
+//				}
```
Remove?

edalex-ian reviewed (approved):
Looks good @PenghaiZhang , just need to remove some left over commented out code. ;)

Once that is done, I reckon let's merge it.
